jenkins_pipelines/environments/uyuni-prs-ci-tests

```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env groovy
 
-node('pull-request-test') {
+node('pull-request-test-nue') {
```

Comment:
We don't care too much where this node is located and this is only used to trigger the pipeline. But.. see below.